src/meshes/t8code_mesh.jl

```diff
@@ -810,32 +972,32 @@ function count_interfaces(mesh::T8codeMesh)
     local_num_mpi_conform = 0
     local_num_mpi_mortars = 0
 
-    visited_global_mortar_ids = Set{UInt64}([])
+    visited_global_mortar_ids = Set{UInt128}([])
 
-    max_level = t8_forest_get_maxlevel(mesh.forest) #UInt64
-    max_tree_num_elements = UInt64(2^ndims(mesh))^max_level
+    max_level = t8_forest_get_maxlevel(forest) #UInt64
```

Comment:
Should the comment indicate that `max_level` is an unsigned?

Reply:
Probably. Do not remember anymore, why I put that comment there. Removed it.